services/bundlejs/bundlejs-package.tester.js

```diff
@@ -0,0 +1,33 @@
+import { isFileSize } from '../test-validators.js'
+import { createServiceTester } from '../tester.js'
+export const t = await createServiceTester()
```

Comment:
Could we add one test cases that uses `.replyWithError()` https://github.com/nock/nock#replying-with-errors to test the `ETIMEDOUT` case

Reply:
Don't know how to test it if the result is permanently cached. Maybe to fake one but I doubt if it's useful.

Reply:
Found a way to test it.

Reply:
Nope. Still unable to test this.